Finmer.Editor/FormMain.cs

```diff
@@ -281,6 +281,19 @@ private bool SaveAs()
             // Present a save dialog
             if (dlgSave.ShowDialog() != DialogResult.OK)
                 return false;
+                
+            // Pull the directory we're saving in, see if there's any other finmer projects already there.
+            string save_path = Path.GetDirectoryName(dlgSave.FileName);
+            string[] file_entries = Directory.GetFiles(save_path, "*.fnproj");
+            if (file_entries.Length >= 1 && !file_entries[0].Equals(dlgSave.FileName))
```

Comment:
The order of the files in the array generated by `Directory.GetFiles` is undefined, so only checking the first element is not enough. You'll need to check if the target file name is anywhere in the array.

Here's an easy solution, which will check if there is at least one file name in the list that does **not** match the user-selected file:
```suggestion
            if (file_entries.Any(name => !name.Equals(dlgSave.FileName)))
```